doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -199,6 +199,8 @@ nRF Desktop
 
   * The :ref:`nrf_desktop_dfu` automatically enables 8-bit write block size emulation (:kconfig:option:`CONFIG_SOC_FLASH_NRF_EMULATE_ONE_BYTE_WRITE_ACCESS`) to ensure that update images with sizes unaligned to word size can be successfully stored in the internal FLASH.
     The feature is not enabled if the MCUboot bootloader is used and the secondary slot is placed in an external FLASH (when :kconfig:option:`CONFIG_PM_EXTERNAL_FLASH_MCUBOOT_SECONDARY` is enabled).
+  * The :ref:`nrf_desktop_ble_latency` uses low latency for the active Bluetooth connection in case of the SMP transfer event regardless of the event submitter module.
+    Previously, the module enabled the low latency for connection only for the :ref:`caf_ble_smp` submitter.
```

Comment:
```suggestion
    Previously, the module enabled the low latency only for the connection of the :ref:`caf_ble_smp` submitter.
```
If the meaning is affected - please disregard this suggestion.

Reply:
thanks, I rephrased this sentence as the meaning seemed to change after your suggestion. Hopefully, it sounds better now. Please take another look

Reply:
looks great, thanks!